data/json/mapgen/town_hall.json

```diff
@@ -75,6 +75,126 @@
+        "***v   hc    ch   v**********v  h  v  v  h  v***",
+        "***|    c    c    |**********|||||||  |||||||***",
+        "***|o  hc    ch   ||||vvvv||||  h  |  |  h  |***",
+        "***|o   cccccc    |op      po| hch v  v hch |***",
+        "***|o    h  h    h|o?? tt ??o| hch v  v hch |***",
+        "***|           hcc|o?  tt  ?o|     +  +  h  |***",
+        "***|    Y   r  hcc|o?? tt ??o| cccc|  |Y    |***",
+        "***|||||||++|||||||o        o|||||||++|||||||***",
+        "***|@i#p          |          | cBBc      #i@|***",
+        "***vii#                                  #iiv***",
+        "***vii#                                  #iiv***",
+        "***|@i#      cBBc |         r|          p#i@|***",
+        "***||||||=!!=||||||          |||||||++|||||||***",
+        "***|<BBBBr   BBBB |###    ###|r   |p  p|   Y|***",
+        "***|              |<<      <<| xd =    = cc |***",
+        "***v BBBB    BBBB |<<      <<| hd !    ! d hv***",
+        "***vp            p|||||++||||| cc =    = dx v***",
+        "***|##############|tthY  Y  h|Y  p|    |  r |***",
+        "***|~~~~~~~~~~~~~~|tt ____   ||||||   6||||||***",
+        "***|~~~~~~~~~~~~~~|hh ____ h |iiii=    Yxccd|***",
+        "***v~~~~~~~~~~~~~~|tt ____   |i<ii!      h dv***",
+        "***v~~~~~~~~~~~~~~|tt ____tt |iiii=    #d hpv***",
+        "***|~~~~~~~~~~~~~~|hh ____tt ||||||    #dccx|***",
+        "***|~~~~~~~~~~~~~~|h  ____   |Y   +    #####|***",
+        "***|~~~~~~~~~~~~~~| h ____  h|ph x|    Yxccd|***",
+        "***v~~~~~~~~~~~~~~|   ____ hh|ddcc|       hdv***",
+        "***v~~~~~~~~~~~~~~|   ____   ||||||    #c hpv***",
+        "***|~~~~~~~~~~~~~~|   ____  h|x h +    #cddx|***",
+        "***|~~~~~~~~~~~~~~| tt____   |d  Y|    #####|***",
+        "***|~~~~~~~~~~~~~~| tt       |dccp|    Yddcc|***",
+        "***v~~~~~~~~~~~~~~|||||++||||||||||     p hxv***",
+        "***v~~~~~~~~~~~~~~|,.#''''#.,|'''+         xv***",
+        "***|~~~~~~~~~~~~~~|..#''''#..|'>'|||++|||h d|***",
+        "***|~~~~~~~~~~~~~~|###''''###|'''|Y   po|ccd|***",
+        "***||v||~~~~~~||v||333ssss333||+||     o||v||***",
+        "*******|~~~~~~|ssssssssssssssssss| ddcco|*******",
+        "*******|~~~~~~|ssssssssssssssssssv  ? xov*******",
+        "*******||vvvv||##################||vvvv||*******",
+        "************************************************",
+        "************************************************",
+        "************************************************",
+        
+      ],
+      "palettes": [ "town_hall_palette" ],
+      "place_monsters": [
+        { "monster": "GROUP_ZOMBIE", "x": [ 24, 47 ], "y": [ 4, 23 ], "density": 0.35 },
+        { "monster": "GROUP_ZOMBIE", "x": [ 24, 47 ], "y": [ 24, 47 ], "density": 0.35 },
+        { "monster": "GROUP_ZOMBIE", "x": [ 24, 47 ], "y": [ 48, 71 ], "density": 0.35 },
+        { "monster": "GROUP_ZOMBIE", "x": [ 24, 47 ], "y": [ 72, 95 ], "density": 0.3 }
+      ]
+    }
+  }
```

Comment:
Shouldn't there be a coma here?

Reply:
I tried it both with and without a comma at the end of each block of the map. And it didn't affect whether it passed the linter.
Unless you are referring to another place?

I left it at off and tried changing a few other minor things that didn't fix it.
I realized I forgot to remove the definitions indicating the duplicate roof tiles So I already removed those.